templates/kubernetes/managed.html

```diff
@@ -3,32 +3,51 @@
         <p>Focus on your business while Canonical builds and manages your Kubernetes clusters. Take advantage of your on-premises assets and leverage public clouds for your multi-cloud strategy.</p>
-        <p>Canonical will set-up and operate your Kubernetes cluster for you, with in-house experts available at all times to stand-up and scale your cluster. And, when you're ready, we will hand over operational control.</p>
-        <ul>
-          <li>Fully managed Kubernetes on your bare metal, VMWare, OpenStack or any public cloud</li>
-          <li>Built in one to three weeks using a proven reference architecture with our <a class="p-link--inverted" href="/kubernetes">Kubernetes consulting packages</a></li>
-          <li>Customise the architecture in iterations based on workloads</li>
-          <li>Enterprise <abbr title="Service level agreement">SLA</abbr> for the infrastructure, upgrades included</li>
-          <li>Monitoring and log management integration</li>
-        </ul>
         <div>
           <a href="https://assets.ubuntu.com/v1/8db65a1e-Kubernetes-for-the-Enterprise.pdf" class="p-button--positive">Read the datasheet</a>
-
-          <a href="/kubernetes/contact-us?product=kubernetes-managed" class="p-button--neutral js-invoke-modal">Talk to us</a>
         </div>
       </div>
-      <div class="col-5 u-align--center u-hide--small">
-        <img src="https://assets.ubuntu.com/v1/4f7c6084-cloud-key.svg" width="378" alt="Kubernetes Managed Service Provider" />
+      <div class="col-6 u-align--center u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/91154a9f-Managed+Kubernetes.svg"
+        width="378" alt="Kubernetes Managed Service Provider" style="height:
+        200px;"/>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--light">
+    <div class="row">
+      <div class="u-fixed-width">
+        <p>Canonical will set-up and operate your Kubernetes cluster for you, with
+        in-house experts available at all times to stand-up and scale your cluster.
+        And, when you're ready, we will hand over operational control. </p>
+        <ul class="p-list is-split">
+          <li class="p-list__item is-ticked">Fully managed Kubernetes on your
+            bare metal, VMWare, OpenStack or any public cloud</li>
+          <li class="p-list__item is-ticked">Built in one to three weeks using
+            a proven reference architecture with our <a
+              href="/kubernetes">Kubernetes consulting
+              packages</a></li>
+          <li class="p-list__item is-ticked">Customise the architecture in iterations based on workloads</li>
+          <li class="p-list__item is-ticked">Enterprise SLA for the
+            infrastructure, upgrades included</li>
+          <li class="p-list__item is-ticked">Monitoring and log management
+            integration</li>
+        </ul>
+        <a
+          href="/kubernetes/contact-us?product=kubernetes-managed">Talk to us&nbsp;&rsaquo;</a>
```

Comment:
One question - should this trigger the modal?

Reply:
@sowasred2012 good question. On the live page the `Talk to us` button triggers the modal. Perhaps this should do the same? @pmahnke what do you think?